packit_service/worker/handlers/testing_farm.py

```diff
@@ -77,6 +78,7 @@
 @reacts_to(PushGitlabEvent)
 @reacts_to(MergeRequestGitlabEvent)
 @reacts_to(AbstractPRCommentEvent)
+@reacts_to(AbstractIssueCommentEvent)
```

Comment:
we want to react only to pull request comments for running the Testing Farm

Reply:
I understood that some of the users want to trigger it from the issues. Is that something we will deal with in another pr? Or I just misunderstood it?

Reply:
we allow retriggering from issues for the [`propose_downstream` job](https://packit.dev/docs/configuration/#propose_downstream)